dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/Tracer.kt

```diff
@@ -22,6 +27,29 @@ import java.util.Properties
  */
 class Tracer internal constructor(config: Config, writer: TraceWriter) : DDTracer(config, writer) {
 
+    private val random: Random = SecureRandom()
+
+    // region Tracer
+
+    override fun buildSpan(operationName: String?): DDSpanBuilder {
+        // On Android, the same zygote is reused for every single application,
+        // meaning that the ThreadLocalRandom reuses the same exact state,
+        // resulting in conflicting TraceIds.
+        // To mitigate this, we recompute our own trace id and override it here
+        val parentContext = activeSpan()?.context() ?: ExtractedContext(
+            BigInteger(63, random),
```

Comment:
What is this `63`? What about extracting it to a named constant?

Reply:
This is the value used by the `dd-trace-ot` implementation, my guess is we want a positive long value (so we only fill the first 63 bits)

Reply:
Done